src/config/config.js

```diff
@@ -163,14 +169,14 @@ const config = convict({
     format: String,
     sensitive: true,
     env: 'AZURE_CLIENT_SECRET',
-    default: ''
+    default: 'test_value'
   },
   oidcWellKnownConfigurationUrl: {
     doc: 'OIDC .well-known configuration URL',
     format: String,
     env: 'OIDC_WELL_KNOWN_CONFIGURATION_URL',
     default:
-      'https://login.microsoftonline.com/6f504113-6b64-43f2-ade9-242e05780007/v2.0/.well-known/openid-configuration'
```

Comment:
This is the infra-dev configuration. It is set in cdp-app-config. We mostly use the stub so setting to this instead

Reply:
updated